vortex-api/src/main/java/mil/army/usace/hec/vortex/VortexGrid.java

```diff
@@ -168,20 +194,33 @@ public VortexGridBuilder dataType (final VortexDataType dataType) {
             return this;
         }
 
-        public VortexGridBuilder dataType (final String dataTypeString) {
-            VortexDataType type = VortexDataType.fromString(dataTypeString);
-            return dataType(type);
+        public VortexGridBuilder vortexVariable (final VortexVariable vortexVariable) {
+            this.variable = vortexVariable;
+            return this;
         }
 
         public VortexGrid build() {
             // Set data type if it has not been set by the builder arg
             if (dataType == null) {
```

Comment:
Is it safe to automatically `inferDataType` if the `dataType` was not specified for a `VortexGrid`?

Reply:
I'm a little be hesitant with the data type field because it can easily conflict with other state variables, e.g. start and end times and units. Should `inferDataType` be called from the getter so we don't have to store this state and risk a conflict?